### PR TITLE
fix: Fixes text overflow issue with notification content

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://github.com/chatwoot/chatwoot/issues/5030

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

**Screenshot**

**Before**

Screenshot 2022-07-15 at 2 45 49 PM

**After**

Screenshot 2022-07-15 at 2 45 39 PM

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
           :key="notificationItem.id"
           @click="() => onClickNotification(notificationItem)"
         >
-          <td>
+          <td class="notification-wrap">
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue:18]: Should we use text-truncate?

Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue]: Done

File: app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
           :key="notificationItem.id"
           @click="() => onClickNotification(notificationItem)"
         >
-          <td>
+          <td class="text-truncate">
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue:18]: @iamsivin Can you please post a screenshot after this change?

Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue:18]: @pranavrajs Same as above. This issue only occurs in links or a sentence with no space. Not showing 

image

File: app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue [hunk 3]
```diff
@@ -15,7 +15,7 @@
...
             <div class="flex-view notification-contant--wrap">
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue:19]: @iamsivin On testing, it seems that it is better to put text-truncate here.

Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/notifications/components/NotificationTable.vue]: Updated the PR

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**